src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetFormatTests.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using System;
+using System.Linq;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+public class DotNetFormatTests
+{
+    const string VerifyNoChangesTestDirectoryName = "VerifyNoChanges";
+    const string FormatTestDirectoryName = "FormatTest";
+    const string UnformattedDirectoryName = "unformatted";
+    const string SolutionDirectoryName = "solution";
+    const string TestFileName = "Test.cs";
+
+    private ITestOutputHelper _outputHelper { get; }
+    private DotNetHelper _dotNetHelper { get; }
+    private string _formatTestRootDirectory { get; }
+
+    public DotNetFormatTests(ITestOutputHelper outputHelper)
+    {
+        this._outputHelper = outputHelper;
+        this._dotNetHelper = new DotNetHelper(outputHelper);
+        this._formatTestRootDirectory = Path.Combine(Directory.GetCurrentDirectory(), "format");
+    }
+
+    /// <Summary>
+    /// Create a new project and verify that it doesn't need to be formatted.
+    /// </Summary>
+    [Fact]
+    public void DotNetFormatVerifyNoChanges()
```

Comment:
Given this is in the scope of DotNetFormatTests, I would suggest renaming to just `VerifyNoChanges`. 

Comment applies to multiple places.